models/aave/optimism/aave_optimism_sources.yml

```diff
@@ -30,12 +31,110 @@ sources:
           description: "Variable borrow index value of the reserve"
         - name: variableBorrowRate
-          description: "Variable borrow rate value of the reserve"   
+          description: "Variable borrow rate value of the reserve"     
+
+      - name: Pool_evt_Supply
+        loaded_at_field: evt_block_time
+        description: "Contians all the transactions involving deposits for aave v3 on optimism"
+        columns:
+        - &amount
+          name: amount
+          description: "The token transafer amount"       
+        - *contract_address 
+        - *evt_block_number
+        - *evt_block_time
+        - *evt_index
+        - *evt_tx_hash 
+        - name: onBehalfOf
+          description: "The user wallet address if the token is deposited on behalf of another wallet"
+        - name: referralCode
+          description: "Deposit referral number"  
+        - *reserve
+        - &user
+          name: user
+          description: "The user wallet address"
+
+      - name: Pool_evt_Withdraw
+        loaded_at_field: evt_block_time
+        description: "Contians all the transactions involving withdrawals for aave v3 on optimism"
+        columns:
+        - *amount       
+        - *contract_address 
+        - *evt_block_number
+        - *evt_block_time
+        - *evt_index
+        - *evt_tx_hash 
+        - *reserve
+        - name: to
+          description: "Wallet address where the tokens are withdrawn and transferred"
+        - *user
+
+      - name: Pool_evt_LiquidationCall
+        loaded_at_field: evt_block_time
+        description: "Contians all the transactions involving liquidation for aave v3 on optimism"
+        columns:
+        - name: collateralAsset
+          description: "Token address of the collateral asset"      
+        - *contract_address 
+        - name: debtAsset
+          description: "Token address of the debt asset"
+        - name: debtToCover
+          description: "Amount of debt to be covered through liquidation"  
+        - *evt_block_number
+        - *evt_block_time
+        - *evt_index
+        - *evt_tx_hash 
+        - name: liquidatedCollateralAmount
+          description: "Amount of liquidated collateral asset"  
+        - name: liquidator
+          description: "Liquidator wallet address"
+        - name: receiveAToken
+          description: "Shows if the liquidator will receive atokens or not"
+        - *user
+
+      - name: Pool_evt_Borrow
+        loaded_at_field: evt_block_time
+        description: "Contians all the transactions involving borrows for aave v3 on optimism"
+        columns:
+        - *amount     
+        - name: borrowRate
+          description: "The rate at which the tokens are borrowed"
+        - *contract_address 
+        - *evt_block_number
+        - *evt_block_time
+        - *evt_index
+        - *evt_tx_hash 
+        - name: interestRateMode
+          description: "The borrowing mode of the debt"

```

Comment:
would be good to show examples of different mode

Reply:
Added